modules/markup/html.go

```diff
@@ -70,9 +70,6 @@ var (
 // CSS class for action keywords (e.g. "closes: #1")
 const keywordClass = "issue-keyword"
 
-// regexp for full links to issues/pulls
-var issueFullPattern *regexp.Regexp
```

Comment:
I would propose a sync.Once loock

Reply:
oh no you dont need to

Reply:
Probably an antipattern but init() would also work, wouldn't it?